sydent/sydent.py

```diff
@@ -176,6 +178,10 @@ def sighup(signum, stack):
             self.cfg.get('userdir', 'userdir.allowed_homeservers', '')
         ))
 
+        self.invites_validity_period = parse_duration(
+            self.cfg.get('general', 'invites.validity_period', 0),
```

Comment:
I don't think the third parameter does what you think it does: https://docs.python.org/2.7/library/configparser.html#ConfigParser.ConfigParser.get